cis194/week2/zehua/src/Exercises.hs

```diff
@@ -0,0 +1,23 @@
+module Exercises where
+
+import Log
+
+-- ex1
+createLogMessage :: MessageType -> String -> [String] -> LogMessage
```

Comment:
This is a partial function here. As per @shineli1984 we need to _avoid partial function at any cost_.

Reply:
how do you tell from the type of the function that this is a partial function?